frontend/src/App.css

```diff
@@ -1,3 +1,7 @@
+html {
+  font-family: Inter, Arial;
```

Comment:
i'd recommend the following very long font-family list to include most system fonts:

`font-family: Inter, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;`